### PR TITLE
Adding a created_at method

### PR DESCRIPTION
**Pull request by IMhide**
Hi

Since I needed the card creation time. Instead of a dirty override to add the method and maybe some other pepole need this method I added it. Even if it's not in the JSON you can find it in the ID.
(http://help.trello.com/article/759-getting-the-time-a-card-or-board-was-created) 

I only added it on the Card object this method should work with the Board object.

### CODE REVIEW

**Comment by joneslee85** [comment]
Please revert the version bump change. I will do it when I prepare for new release.

**Comment by IMhide** [comment]
The version bump have been removed

**Comment by joneslee85** [comment]
@IMhide can you please resolve the conflicts?

File: spec/card_spec.rb [hunk 1]
```diff
@@ -220,6 +220,11 @@ module Trello
       it "gets its pos" do
         expect(card.pos).to_not be_nil
       end
+
+      it "gets its creation date" do
```
Comment (joneslee85) [inline spec/card_spec.rb]: can you please assert against a static value? I could not tell what value it would produce.

**Comment by IMhide** [comment]
is it ok like this ?

**Comment by joneslee85** [comment]
yup, can you please not assert against nil? How about assert against the true value instead?

**Comment by IMhide** [comment]
I'm not asserting against `nil`.
I'm rescuing from `Time.at` in case of `id` is `nil`.
I'm trying to make this method looks like ActiveRecord created_at attribute. With ActiveRecord if you don't have an `id` `created_at` is nil

**Comment by IMhide** [comment]
Is that what you wanted ?

**Comment by joneslee85** [comment]
nice, will merge on green

**Pull request merged**